pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -24,13 +25,19 @@ stdenv.mkDerivation {
 
   buildInputs = [
     bison flex perl pkgconfig libpcap lua5 openssl libgcrypt gnutls
-    geoip libnl c-ares python libcap glib
+    geoip libnl c-ares python libcap glib zlib
   ] ++ optional withQt qt4
     ++ optional withGtk gtk;
 
   patches = [ ./wireshark-lookup-dumpcap-in-path.patch ];
 
+  makeFlags = [
+    "DESTDIR=$(out)"
+    "PREFIX="
+  ];
+
   configureFlags = "--disable-usr-local --disable-silent-rules --with-ssl"
+    + " --prefix=/"
```

Comment:
The `--prefix=/` flag here looks very suspicious. I have a hard time imagining that this setting configures a build that generates working binaries.